app/uk/gov/hmrc/gform/validation/ValidationResult.scala

```diff
@@ -63,6 +63,12 @@ case class ValidationResult(
       case (_, FieldOk(formComponent, cv)) if formComponent.isReferenceNumber =>
         val cvUpd: String = cv.replace(" ", "")
         FieldOk(formComponent, cvUpd)
+      case (_, FieldOk(formComponent, cv)) if formComponent.isSortCode =>
+        val cvUpd: String = cv
+          .replaceAll("[^0-9]", "")
+          .grouped(2)
+          .mkString("-")
```

Comment:
Saving sortcode with dashes would break our bank account check calls. We need to store sortcode as 6 digits and then format it when we want to replay it back to the user.